utils/src/treev2/quickPickWizard/FindByIdQuickPickStep.ts

```diff
@@ -50,49 +49,20 @@ export class FindByIdQuickPickStep<TNode extends types.FindableByIdTreeNode, TCo
         }
     }
 
-    protected override isIndirectPick(node: TNode): boolean {
-        if (isFindableByIdTreeNodeV2(node)) {
-            if (node.quickPickOptions.isLeaf) {
-                return false;
-            }
-
-            return this.pickOptions.id.startsWith(node.id);
-        } else {
-            if (!isAzExtParentTreeItem(node)) {
-                return false;
-            }
-
-            return this.pickOptions.id.startsWith(node.fullId);
+    protected override isIndirectPick(node: vscode.TreeItem): boolean {
+        if (!node.collapsibleState) {

```

Comment:
It's obviously a super minor code duplication, but we could replace this part of this with `super.isIndirectPick(node)`.

e.g.:
```typescript
return super.isIndirectPick(node) && node.id && this.pickOptions.id.startsWith(node.id);
```

Also, you have `node.id.startsWith(node.id)`, you meant `this.pickOptions.id.startsWith(node.id)`

Reply:
I don't think we can use `super.isIndirectPick` here since the parent class is `GenericQuickPickStep`, right?

Reply:
You're right. I spaced and though this was inheriting from `ContextValueQuickPickStep`, due to file alphabetization not matching inheritance structure.